core/controllers/collection_viewer.py

```diff
@@ -93,12 +93,16 @@ def get(self, collection_id):
                     allow_invalid_explorations=False))
         except Exception as e:
             raise self.PageNotFoundException(e)
+        collection_summary = \
```

Comment:
Never end a line with a backslash. If there's a space after the backslash, it won't get noticed. See the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#python).

Instead:
```
        collection_summary = (
            summary_services.get_displayable_collection_summary_dicts_matching_ids(
                collection_id))
```